docs/components/icon.md

```diff
@@ -23,21 +23,62 @@ following list. It tells KIcon which svg to render.
         class="mb-4"
         @click="toggle">Toggle viewbox {{ isToggled ? 'off' : 'on' }}</KButton>
       <div class="icon-row">
-        <div
-          v-for="icon in $icons"
-          class="icon-cell"
-          :class="{ hasBg: isToggled }">
-          <KIcon
+        <div v-for="icon in $icons">
+          <div v-if="!stateIcons.includes(String(icon))"
+            class="icon-cell"
+            :class="{ hasBg: isToggled }"
+          >
+            <KIcon
             size="24"
             :icon="icon" />
-          <span>{{ icon }}</span>
+            <span>{{ icon }}</span>
+          </div>
         </div>
       </div>
     </div>
   </KToggle>
 </div>
 &nbsp;
 
+#### state icons
+
+There are 5 icons: `stateConfigure`, `stateGruceo`, `stateNoData`, `stateNoSearchResults`, `stateUpload` that are used in `KEmptyState`.
```

Comment:
Let's reword this a bit so that we don't have to list out any future new icons that get added
```suggestion
The `state`-prefixed icons shown below are styled differently than our normal icons listed above and are meant to be used inside instances of `KEmptyState` where they appear larger in the UI.
```